hera_sim/simulate.py

```diff
@@ -57,26 +57,31 @@ def __init__(self, data=None, defaults_config=None, **kwargs):
         # actually initialize the UVData object stored in self.data
         self._initialize_data(data, **kwargs)
 
-    @cached_property
+    @property
     def antpos(self):
         # TODO: docstring
         """
         """
         antpos, ants = self.data.get_ENU_antpos(pick_data_ants=True)
         return dict(zip(ants, antpos))
 
-    @cached_property
+    @property
     def lsts(self):
         # TODO: docstring
         return np.unique(self.data.lst_array)
 
-    @cached_property
+    @property
     def freqs(self):
         # TODO: docstring
         """Frequencies in GHz
         """
         return np.unique(self.data.freq_array) / 1e9
 
+    @property
+    def times(self):
+        """Return unique simulation times."""
+        return np.unique(self.data.time_array)
+
     # XXX begin methods intended for user interaction XXX
```

Comment:
remove this comment

Reply:
OK. There's a similar comment further down; I'll remove that too.